contrib/buildrefactor/src/python/pants/contrib/buildrefactor/peek.py

```diff
@@ -0,0 +1,44 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+#./pants peek --path=src/scala/org/pantsbuild/zinc/analysis --target-name=zinc-analysis
+#./pants peek --path=src/scala/org/pantsbuild/zinc/extractor --target-name=zinc-extractor --line_number=True
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.task.task import Task
+from pants.contrib.buildrefactor.buildozer import Buildozer
+
+
+class Peek(Task):
+    """Peek a build file for a specified target
+
+    line_number: optional flag to print the starting line of the target if found
+    """
+
+    @classmethod
+    def register_options(cls, register):
+        super(Peek, cls).register_options(register)
+
+        register('--line_number', help='Prints the starting line number of the named target.', type=bool, default=False)
```

Comment:
nit: specify this as `--line-number`

Also, you can pass a bool flag as just the flag name, `--line-number`. You don't need the `=true`, so it'd be good to update the examples using that syntax.